src/components/data-table/migrate-to-10.x.md

```diff
@@ -1,20 +1,79 @@
 ## Data Table
 
+### HTML
+
+Aside from using new icons from the [`carbon-elements`](https://github.com/IBM/carbon-elements) package, there are several structural changes and new selectors in the Data Table component. Please see the new structure below and reference the [data table page](https://next.carbondesignsystem.com/components/data-table/code) in our site to copy the specified new markup.
+
+React and other framework variants should reflect the changes automatically.
```

Comment:
There's some changes that folks are going to have to make (toolbar/search related)

Reply:
@vpicone What should I add specifically regarding that?